src/dataflow-types/src/storage.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright Materialize, Inc. and contributors. All rights reserved.
```

Comment:
Why `storage.rs` over `source.rs`?

Reply:
I accidentally included this file, oops. Removed